### PR TITLE
Add missing fields

### PR DESCRIPTION
**Pull request by mruwnik**

### CODE REVIEW

File: align_data/distill/distill.py [hunk 1]
```diff
@@ -1,94 +1,32 @@
...
-        body = markdown_text
-
-        # pull the ol with class references out of the soup
-        references = soup.find("ol", {"class": "references"})
-        if references:
-            # for each reference li, get the the span with the class title
-            references = [
-                {"title": reference.find("span", {"class": "title"}).text}
-                for reference in references.find_all("li")
+class Distill(RSSDataset):
```
Comment (mruwnik) [inline align_data/distill/distill.py:7]: dunno about this. It slows things down, but fetches the contents + doesn't need to keep the raw html in repo

Comment (ccstan99) [reply align_data/distill/distill.py:7]: The distill.pub website will no longer be updated, so keeping contents on gdrive is an option. Either way, the raw html shouldn't be in the repo.

File: align_data/alignment_newsletter/alignment_newsletter.py [hunk 2]
```diff
@@ -35,21 +33,31 @@ def process_entry(self, row):
...
+            if not v or pd.isna(v):
+                return None
+            if cast:
+                return cast(v)
+            return v
+
         return DataEntry({
-            "url": "https://rohinshah.com/alignment-newsletter/",
-            "source": self.name,
+            "url": handle_na(row.URL) or 'n/a',
```
Comment (ccstan99) [inline align_data/alignment_newsletter/alignment_newsletter.py:47]: Looks like `url` is the source being summarized, correct? Just be sure when creating the IDs, these should be modified so it doesn't clash with the hashed url from the source, i.e. arxiv, alignment forum, lesswrong, etc.

**Review by ccstan99 — APPROVED** [review]
High level it looks fine but I'm feeling a bit Machievllian. It might be easier to just review the scraped out dataset after you've pushed it to HF.

File: align_data/alignment_newsletter/alignment_newsletter.py (hunk 2 above)
Comment (mruwnik) [reply align_data/alignment_newsletter/alignment_newsletter.py:47]: yes - good point

**Pull request merged**